src/ElasticSearch.php

```diff
@@ -183,14 +183,13 @@ protected function buildResultSet($elasticResult = []): ResultSet
         $total = 0;
         if ($elasticResult && isset($elasticResult['hits'])) {
             foreach ($elasticResult['hits']['hits'] as $document) {
-                $uris[] = $document['_id'];
+                $uris[] = array_merge($document['_source'], ['id' => $document['_id']]);
```

Comment:
```suggestion
                $document['_source']['id'] = $document['_id']
                
                $uris[] = $document['_source'];
```

It took me a while you just wanted to add the id, maybe simplify it?